guest.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+const cp = require('child_process')
+const argv = process.argv.slice(process.argv[1] === __filename ? 2 : 1)
+const files = []
+
+// Would prefer /app but that's owned by root in the prebuild images
+const cwd = '/home/node/app'
+
+// Copy host files to working directory
+for (const file of files) {
+  const a = path.join('/input', file)
+  const b = path.join(cwd, file)
+
+  mkdirp(path.dirname(b))
+
+  fs.copyFileSync(a, b, fs.constants.COPYFILE_EXCL)
+  fs.chmodSync(b, 0o644)
+}
+
+// Use node_modules of host to avoid a second install step
+fs.symlinkSync('/input/node_modules', path.join(cwd, 'node_modules'))
+
+// TODO: bundle this
+const tar = require(path.join(cwd, 'node_modules', 'tar-fs'))
```

Comment:
`browserify --bare` or `noderify`, and then mount `bundle.js` as a volume?

Or, don't use `tar-fs`, but spawn `tar`?

Or, `require.resolve('tar-fs', { paths: .. })` (which would not work if `prebuildify-cross` is installed globally)?